spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/BaseFileRewriteCoordinator.java

```diff
@@ -72,18 +70,12 @@ public void clearRewrite(Table table, String fileSetId) {
 
   public Set<String> fetchSetIds(Table table) {
     return resultMap.keySet().stream()
-        .filter(e -> e.first().equals(tableUUID(table)))
+        .filter(e -> e.first().equals(Spark3Util.tableUUID(table)))
```

Comment:
Nit: I think it would be a bit cleaner just to import Spark3Util.tableUUID and then just use tableUUID here (the diff on line 73 and other places would essentially go away in favor of just a new import statement)

Reply:
Yeah, I renamed to `baseTableUUID` and I am not sure about the guidelines on static import. Some place we use it and some place we don't. So, I left it as it is.